<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCPNCodeSetGUIDlg</class>
 <widget class="QDialog" name="TCPNCodeSetGUIDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DDTBBE_CodFrmSynSetGUIDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox
{
	border-width:2px;
	border-style:solid;
	border-color:gray;
	margin-top:1ex;
}

QGroupBox::title
{
	subcontrol-origin:margin;
	subcontrol-position:top left;
	left:10px;
	padding: 0px;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>891</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>主接收伪码</string>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>851</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>上行伪码</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_UpTCtrlPsePhase2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>143</y>
       <width>100</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_UpTCtrlPseOriginPoly2">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>67</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>遥控支路伪码本源多项式2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_UpTCtrlPsePhase1">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>103</y>
       <width>100</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_UpTCtrlPseOriginPoly1">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>遥控支路伪码本源多项式1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_UpTCtrlPsePhase2">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>143</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>遥控支路伪码初相2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_UpTCtrlPseOriginPoly1">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>100</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_UpTCtrlPsePhase1">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>103</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>遥控支路伪码初相1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_UpTCtrlPseOriginPoly2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>67</y>
       <width>100</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Set">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Modify">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>更改</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_targat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>891</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>目标选择</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Target1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>16</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>目标1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Target2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>16</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>目标2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Target3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>16</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>目标3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Target4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>16</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>目标4</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
